benches/json.rs

```diff
@@ -118,7 +142,7 @@ fn json(input: &str) -> IResult<&str, JsonValue> {
 
 fn json_bench(c: &mut Criterion) {
   let data = "  { \"a\"\t: 42,
-  \"b\": [ \"x\", \"y\", 12 ] ,
+  \"b\": [ \"x\", \"y\", 12 ,\"\\u2014\", \"\\uD83D\\uDE10\"] ,
```

Comment:
Not sure I would include this in the benchmark, as I don't think it's a very common case.

Reply:
Fair. I was just thinking to cover all the paths.